docroot/modules/custom/mass_entity_usage/mass_entity_usage.module

```diff
@@ -45,3 +46,86 @@ function mass_entity_usage_queue_info_alter(&$queues) {
     unset($queues['entity_usage_tracker']['cron']);
   }
 }
+
+/**
+ * Implements hook_module_implements_alter().
+ */
+function mass_entity_usage_module_implements_alter(&$implementations, $hook) {
+  $overrides = [
+    'entity_insert',
+    'entity_update',
+    'entity_predelete',
+    'entity_translate_delete',
+    'entity_revision_delete',
+  ];
+
+  // Completely take over entity_usage hooks.
```

Comment:
Not a common practice, but it is working reliably. Previously, we had a patch that added conditional logic to the entity usage hooks to use the tracking queue instead of updating entity_usage immediately.

Reply:
This also only occurs if the queue_tracking config is set to true.